requirements-test.txt

```diff
@@ -1,7 +1,18 @@
 -r requirements.txt
-flake8
-moto==0.3.1
-nose==1.3.3
-# unittest v1.0.0 and v1.0.1 have a dependency on linecache2.  linecache2 is
-# currently experiencing syntax errors when you attempt to install it.
-unittest2==0.8.0
+
+flake8==3.7.9 \
+    --hash=sha256:45681a117ecc81e870cbf1262835ae4af5e7a8b08e40b944a8a6e6b895914cfb \
+    --hash=sha256:49356e766643ad15072a789a20915d3c91dc89fd313ccd71802303fd67e4deca
+moto==1.3.14 \
+    --hash=sha256:2b3fa22778504b45715868cad95ad458fdea7227f9005b12e522fc9c2ae0cabc \
+    --hash=sha256:79aeaeed1592a24d3c488840065a3fcb3f4fa7ba40259e112482454c0e48a03a
+nose==1.3.7 \
+    --hash=sha256:9ff7c6cc443f8c51994b34a667bbcf45afd6d945be7477b52e97516fd17c53ac \
+    --hash=sha256:dadcddc0aefbf99eea214e0f1232b94f2fa9bd98fa8353711dacb112bfcbbb2a \
+    --hash=sha256:f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98
+unittest2==1.1.0 \
+    --hash=sha256:13f77d0875db6d9b435e1d4f41e74ad4cc2eb6e1d5c824996092b3430f088bb8 \
+    --hash=sha256:22882a0e418c284e1f718a822b3b022944d53d2d908e1690b319a9d3eb2c0579
+WebTest==2.0.34 \
```

Comment:
👍 this fits better in ``requirements-test.txt``, I hope this means that mozsrc has fixed the "wonky issues with testing dependencies"

Reply:
🤞